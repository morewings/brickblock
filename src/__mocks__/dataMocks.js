export const contributorsMock = [
  {
    address: '1KLzFuKuRqTBXxAJ4ra4UyMB8Ey2XHHhTc',
    currency: 'BTC',
    value: 2330000,
    txid: 'ea57a17b9b21063c9b032d2ce3312aa701e9c2ab399b42cf93848f4702dcdab0',
  },
  {
    address: '853a217dcbad301698f62a979d15dc68558e53ab',
    currency: 'ETH',
    value: 5000000000000000000,
    txid: 'c1b193840595d1b6b2a30f785de6bc2358a18d6140726f8c23022b9db798b633',
  },
  {
    address: 'LSnQyESPeVbbyeZivGLC78ZWj1pinq9MsT',
    currency: 'LTC',
    value: 437031973,
    txid: 'cb039bd814b76e1fd88fad6de5ff2d84f5d3f01e8669fc34a7e11e4b1e8f6317',
  },
  {
    address: '2cea50743963b2de0d1ddc7084981ba05ba3b0c6',
    currency: 'ETH',
    value: 61168830000000000,
    txid: '577168b02c917a09d4256e04f2980f18d57ea11ce96b405574044da9368c23b6',
  },
  {
    address: '19kQT5BhNm55ugD8osKQRzumjU75kWQQsP',
    currency: 'BTC',
    value: 5639300,
    txid: '0d49d28398e5978e6af1df7847bcc1b30ed399742e90334d0460c3e2592a66f4',
  },
  {
    address: '2f23c16e01b0c2bc19178994bb845435f9a708cc',
    currency: 'ETH',
    value: 2854000000000000000,
    txid: 'c67c1ef9ee37788a31fe21f129d8563f119948ed0463b1c307c0c3cacba3679f',
  },
  {
    address: 'b2ce5a893083ae5ec9d4f0a7715984d574c15000',
    currency: 'ETH',
    value: 100000000000000000,
    txid: 'd64dd9aa4f093aec656ad171961ceb317b2e6147f6aadc9425fa9f069a87e609',
  },
  {
    address: '266ebaf5bc6deb5c0b9b40318c97a58a17b648eb',
    currency: 'ETH',
    value: 500000000000000000,
    txid: '4343e557f6f8731ae5f99472e042650834c27de0cd454a77e5fa037ef6a037e7',
  },
  {
    address: 'b7a097895c46e0f471b10cbe34717c3a5c2a658a',
    currency: 'ETH',
    value: 4000000000000000000,
    txid: '1e6da67cf619da05d82f009357b6e0ff1131d0b76789cb9cf0d4620824631576',
  },
  {
    address: '3422VtS7UtCvXYxoXMVp6eZupR252z85oC',
    currency: 'BTC',
    value: 300000,
    txid: '10a8d7cb7f062ebde0f7c39e9735f70772cd76be6485882ebba43227a46abfb9',
  },
  {
    address: '09e5d3f3c4d8ff82a9f697082ea7fa8ba0f70184',
    currency: 'ETH',
    value: 750000000000000000,
    txid: 'eb54633a1ef654853b620516649afec82a04139b112d9a5cd45f66210656582b',
  },
  {
    address: '57b2432819b682018e96d486a7953af257ac7b79',
    currency: 'ETH',
    value: 2000000000000000000,
    txid: '913e253e895691c8e7573bf53ecd9d57e67fe2b17e3ff76ddcb5313e182c3807',
  },
  {
    address: '1Nrb8feDXsUE388qwWY9xhz8xfDRzVw9JD',
    currency: 'BTC',
    value: 5835231,
    txid: 'c6eb75e83abfdee30df0b381227a76e508db72478bda6bee327f3ce81a86230d',
  },
  {
    address: '1HjiFpvcQLq9AQDmxZrjHpfYEVpRY2nVmw',
    currency: 'BTC',
    value: 5773579,
    txid: '9cef76306307636adb3cbadbf68fb21f89ca41127657bc45cace618e9e1e281b',
  },
  {
    address: '1Ldcz86BdD3gSE8cabMJ61hLwxupWQCbiM',
    currency: 'BTC',
    value: 5773579,
    txid: '9cef76306307636adb3cbadbf68fb21f89ca41127657bc45cace618e9e1e281b',
  },
  {
    address: '826278faca81b3150d346823b4ed1ea500ed15d1',
    currency: 'ETH',
    value: 1598200000000000,
    txid: 'c7178c0ef2cb3b8ad92a3f4b62b223a8ccb9588db83c117dff711831d2708940',
  },
  {
    address: '344NfE2kyK5wuUPgkE7AXN9JwLWE84faFv',
    currency: 'BTC',
    value: 3000000,
    txid: 'dac33b953e3c2015b1c394803134a80e80e540406bf83ff28f6815dfb8cb49d7',
  },
  {
    address: '7bc3229b77fbd6bf75fae81453748e15a098ee0b',
    currency: 'ETH',
    value: 394000000000000000,
    txid: '23b7ede790b4c71f78b9033b352185c3e3f1e8ab65ee3b96caf608e924508532',
  },
  {
    address: 'df28b94cc512141c2a02994a5195526e97328a2f',
    currency: 'ETH',
    value: 4996440211000000000,
    txid: '57e4d6ec99f817ae8458cf0cff665113062f7f6be2a53683c995793522fad006',
  },
  {
    address: '7e4538e819dc854b5ae323b83d94269b98388293',
    currency: 'ETH',
    value: 587100909200003300,
    txid: 'a810ce4302bcb78cf7cb66de3fd8048df92ff71edb003b3aab23f9a8d566aeba',
  },
  {
    address: '6cbe1453ee5743108afd12a0b37be088964dcc54',
    currency: 'ETH',
    value: 1299000000000000000,
    txid: '2a8950bc31f2ec81d5772c90b120ec093aca506ab051e75ec426f90f56512df5',
  },
  {
    address: '24dfcc8925b91bc77afd52c3e3286e5f8f4f2445',
    currency: 'ETH',
    value: 300000000000000000,
    txid: '855b2764ccdf262a637c58920d946d68bed75ff7e7dae88421ca7335f2449973',
  },
  {
    address: 'e83dc1d9f5aa8223acb090be5c14877df8c3f71b',
    currency: 'ETH',
    value: 24834800000000000,
    txid: '0ab33f3dcd9dbb233372b61888b245c3c19fe3794bc299d7f2be55f294bb7046',
  },
  {
    address: '1BFYggPKpVNtZmK8V7hHQvQ9FC9LAt61gH',
    currency: 'BTC',
    value: 70000,
    txid: 'ead54c476870f20a3aca491fcf16b13c203f4c95008593865e210da04bdebf9d',
  },
  {
    address: '18hh8V5fK62w13UcZEgabyHfLPc3HLBKCj',
    currency: 'BTC',
    value: 556105,
    txid: '62815ce88ee2660c1e3ae65683254fa61bcb00983e95b42097226a30ff07ed9b',
  },
  {
    address: '1JA9dei6yMxpMWb2RWExu2pB4E3JwpMs5P',
    currency: 'BTC',
    value: 738510,
    txid: '39b1ffaac9468f737f4703cfe71bece39bae934792dd9dc59e36feb30c424d12',
  },
  {
    address: '3757WBguTEpkDT1FMwmFXQMRNmAgWTCks8',
    currency: 'BTC',
    value: 1500000,
    txid: '08f03430265a094ccca78db15eacb7e5b270f641d2a86c3991ba8c38655a054b',
  },
  {
    address: '1CrE4Jn5TCNw22RZq6phM9dF1Ma6Km89Ew',
    currency: 'BTC',
    value: 1500000,
    txid: '08f03430265a094ccca78db15eacb7e5b270f641d2a86c3991ba8c38655a054b',
  },
  {
    address: '1CiFkohNFrSSwJLCrKbYzuh8T5hHi5Q1rQ',
    currency: 'BTC',
    value: 1500000,
    txid: '08f03430265a094ccca78db15eacb7e5b270f641d2a86c3991ba8c38655a054b',
  },
  {
    address: '3422VtS7UtCvXYxoXMVp6eZupR252z85oC',
    currency: 'BTC',
    value: 1349455,
    txid: '114e6bfc5ba97943e02bdd2d0ec26856048699dd53356ff73672737fee8db409',
  },
  {
    address: 'LbTXvopZ7AE67o6bPVpQVfYLXhv52rtME2',
    currency: 'LTC',
    value: 190000000,
    txid: '7b81396570855bc3ce2dee3b5551f02c9d1f8f1a3d8bf8720e7e227d8b887988',
  },
  {
    address: '985077b8a99b2a87a33b8b5e4da8e63ffaa0f8e6',
    currency: 'ETH',
    value: 140000000000000000,
    txid: 'c905b1b9a6cc406711e55f8f4c7a4dc54bbbc0dd4e475c350fa77b937528fcb1',
  },
  {
    address: '18PFNuXwj25Bm2obgL4YM4ACtfuEzmDFVx',
    currency: 'BTC',
    value: 55635,
    txid: '1104bb9dd49424a451f9964222ca1a2ef017cd863acfdc6ab755800e307fa16c',
  },
  {
    address: '171YbBoD8HqyJpWUhUf4AwoWtrYHYqsiRo',
    currency: 'BTC',
    value: 55635,
    txid: '1104bb9dd49424a451f9964222ca1a2ef017cd863acfdc6ab755800e307fa16c',
  },
  {
    address: 'f42686bbc06b8309608ec4d1e46cb4329748634e',
    currency: 'ETH',
    value: 1350000000000000000,
    txid: '8109e8a871ccd7faed096ce3638677dd0cdce6b3184e03b25bbb21fe58228881',
  },
  {
    address: '1D4eY2xAmu8LWmayWahXsn73gscEK18hDf',
    currency: 'BTC',
    value: 1000000,
    txid: 'b2c34a715a4d6ba0b27137a9fbc92cd54603a93929220301cf9e3e3e8fe3784b',
  },
  {
    address: '1Lo459z1FXLXsTkaCDPuTno2TX9EMNJS83',
    currency: 'BTC',
    value: 1000000,
    txid: 'b2c34a715a4d6ba0b27137a9fbc92cd54603a93929220301cf9e3e3e8fe3784b',
  },
  {
    address: '18Xuuvnm6JEjmGUyRFeBVAQSKabZRA3fmG',
    currency: 'BTC',
    value: 1000000,
    txid: 'b2c34a715a4d6ba0b27137a9fbc92cd54603a93929220301cf9e3e3e8fe3784b',
  },
  {
    address: '19v73cyNt8Mi1uXb8CTEX2fhELz3h4Eb8z',
    currency: 'BTC',
    value: 1000000,
    txid: 'b2c34a715a4d6ba0b27137a9fbc92cd54603a93929220301cf9e3e3e8fe3784b',
  },
  {
    address: '1BUu9qgeU2GvofVB4vJcx3oeePkiZTKBpt',
    currency: 'BTC',
    value: 1000000,
    txid: 'b2c34a715a4d6ba0b27137a9fbc92cd54603a93929220301cf9e3e3e8fe3784b',
  },
  {
    address: '175T5CLcUswxupqyFvSC6dEe6PX2bTRLWL',
    currency: 'BTC',
    value: 1000000,
    txid: 'b2c34a715a4d6ba0b27137a9fbc92cd54603a93929220301cf9e3e3e8fe3784b',
  },
  {
    address: '5f029216cb1f5f0622f1029bb7f42a1b061b5bc3',
    currency: 'ETH',
    value: 1000000000000000000,
    txid: '0d4860e8e658d9734f9f21720639d50cec0d5c93a921b5b61ec0dbd0a3e8757c',
  },
];
