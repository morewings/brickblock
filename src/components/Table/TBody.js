import React from 'react';
import PropTypes from 'prop-types';
import TRow from 'components/Table/TRow';

const TBody = props => (
  <tbody>
    {props.content.map((rowData, i) => (
      <TRow
        key={`tableRow-${i}`}
        {...rowData} />
        ))}
  </tbody>
);

TBody.propTypes = {
  content: PropTypes.arrayOf(PropTypes.shape({
    address: PropTypes.string,
    currency: PropTypes.string,
    value: PropTypes.number,
    txid: PropTypes.string,
  })).isRequired,
};

export default TBody;
