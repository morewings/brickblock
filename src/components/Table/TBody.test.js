import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import TBody from 'components/Table/TBody';
import {contributorsMock} from '__mocks__/dataMocks';

describe('components > Table > TBody', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<TBody content={contributorsMock} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
