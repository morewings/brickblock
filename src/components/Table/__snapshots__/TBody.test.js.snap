// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components > Table > TBody renders without crashing 1`] = `
<tbody>
  <TRow
    address="1KLzFuKuRqTBXxAJ4ra4UyMB8Ey2XHHhTc"
    currency="BTC"
    key="tableRow-0"
    txid="ea57a17b9b21063c9b032d2ce3312aa701e9c2ab399b42cf93848f4702dcdab0"
    value={2330000}
  />
  <TRow
    address="853a217dcbad301698f62a979d15dc68558e53ab"
    currency="ETH"
    key="tableRow-1"
    txid="c1b193840595d1b6b2a30f785de6bc2358a18d6140726f8c23022b9db798b633"
    value={5000000000000000000}
  />
  <TRow
    address="LSnQyESPeVbbyeZivGLC78ZWj1pinq9MsT"
    currency="LTC"
    key="tableRow-2"
    txid="cb039bd814b76e1fd88fad6de5ff2d84f5d3f01e8669fc34a7e11e4b1e8f6317"
    value={437031973}
  />
  <TRow
    address="2cea50743963b2de0d1ddc7084981ba05ba3b0c6"
    currency="ETH"
    key="tableRow-3"
    txid="577168b02c917a09d4256e04f2980f18d57ea11ce96b405574044da9368c23b6"
    value={61168830000000000}
  />
  <TRow
    address="19kQT5BhNm55ugD8osKQRzumjU75kWQQsP"
    currency="BTC"
    key="tableRow-4"
    txid="0d49d28398e5978e6af1df7847bcc1b30ed399742e90334d0460c3e2592a66f4"
    value={5639300}
  />
  <TRow
    address="2f23c16e01b0c2bc19178994bb845435f9a708cc"
    currency="ETH"
    key="tableRow-5"
    txid="c67c1ef9ee37788a31fe21f129d8563f119948ed0463b1c307c0c3cacba3679f"
    value={2854000000000000000}
  />
  <TRow
    address="b2ce5a893083ae5ec9d4f0a7715984d574c15000"
    currency="ETH"
    key="tableRow-6"
    txid="d64dd9aa4f093aec656ad171961ceb317b2e6147f6aadc9425fa9f069a87e609"
    value={100000000000000000}
  />
  <TRow
    address="266ebaf5bc6deb5c0b9b40318c97a58a17b648eb"
    currency="ETH"
    key="tableRow-7"
    txid="4343e557f6f8731ae5f99472e042650834c27de0cd454a77e5fa037ef6a037e7"
    value={500000000000000000}
  />
  <TRow
    address="b7a097895c46e0f471b10cbe34717c3a5c2a658a"
    currency="ETH"
    key="tableRow-8"
    txid="1e6da67cf619da05d82f009357b6e0ff1131d0b76789cb9cf0d4620824631576"
    value={4000000000000000000}
  />
  <TRow
    address="3422VtS7UtCvXYxoXMVp6eZupR252z85oC"
    currency="BTC"
    key="tableRow-9"
    txid="10a8d7cb7f062ebde0f7c39e9735f70772cd76be6485882ebba43227a46abfb9"
    value={300000}
  />
  <TRow
    address="09e5d3f3c4d8ff82a9f697082ea7fa8ba0f70184"
    currency="ETH"
    key="tableRow-10"
    txid="eb54633a1ef654853b620516649afec82a04139b112d9a5cd45f66210656582b"
    value={750000000000000000}
  />
  <TRow
    address="57b2432819b682018e96d486a7953af257ac7b79"
    currency="ETH"
    key="tableRow-11"
    txid="913e253e895691c8e7573bf53ecd9d57e67fe2b17e3ff76ddcb5313e182c3807"
    value={2000000000000000000}
  />
  <TRow
    address="1Nrb8feDXsUE388qwWY9xhz8xfDRzVw9JD"
    currency="BTC"
    key="tableRow-12"
    txid="c6eb75e83abfdee30df0b381227a76e508db72478bda6bee327f3ce81a86230d"
    value={5835231}
  />
  <TRow
    address="1HjiFpvcQLq9AQDmxZrjHpfYEVpRY2nVmw"
    currency="BTC"
    key="tableRow-13"
    txid="9cef76306307636adb3cbadbf68fb21f89ca41127657bc45cace618e9e1e281b"
    value={5773579}
  />
  <TRow
    address="1Ldcz86BdD3gSE8cabMJ61hLwxupWQCbiM"
    currency="BTC"
    key="tableRow-14"
    txid="9cef76306307636adb3cbadbf68fb21f89ca41127657bc45cace618e9e1e281b"
    value={5773579}
  />
  <TRow
    address="826278faca81b3150d346823b4ed1ea500ed15d1"
    currency="ETH"
    key="tableRow-15"
    txid="c7178c0ef2cb3b8ad92a3f4b62b223a8ccb9588db83c117dff711831d2708940"
    value={1598200000000000}
  />
  <TRow
    address="344NfE2kyK5wuUPgkE7AXN9JwLWE84faFv"
    currency="BTC"
    key="tableRow-16"
    txid="dac33b953e3c2015b1c394803134a80e80e540406bf83ff28f6815dfb8cb49d7"
    value={3000000}
  />
  <TRow
    address="7bc3229b77fbd6bf75fae81453748e15a098ee0b"
    currency="ETH"
    key="tableRow-17"
    txid="23b7ede790b4c71f78b9033b352185c3e3f1e8ab65ee3b96caf608e924508532"
    value={394000000000000000}
  />
  <TRow
    address="df28b94cc512141c2a02994a5195526e97328a2f"
    currency="ETH"
    key="tableRow-18"
    txid="57e4d6ec99f817ae8458cf0cff665113062f7f6be2a53683c995793522fad006"
    value={4996440211000000000}
  />
  <TRow
    address="7e4538e819dc854b5ae323b83d94269b98388293"
    currency="ETH"
    key="tableRow-19"
    txid="a810ce4302bcb78cf7cb66de3fd8048df92ff71edb003b3aab23f9a8d566aeba"
    value={587100909200003300}
  />
  <TRow
    address="6cbe1453ee5743108afd12a0b37be088964dcc54"
    currency="ETH"
    key="tableRow-20"
    txid="2a8950bc31f2ec81d5772c90b120ec093aca506ab051e75ec426f90f56512df5"
    value={1299000000000000000}
  />
  <TRow
    address="24dfcc8925b91bc77afd52c3e3286e5f8f4f2445"
    currency="ETH"
    key="tableRow-21"
    txid="855b2764ccdf262a637c58920d946d68bed75ff7e7dae88421ca7335f2449973"
    value={300000000000000000}
  />
  <TRow
    address="e83dc1d9f5aa8223acb090be5c14877df8c3f71b"
    currency="ETH"
    key="tableRow-22"
    txid="0ab33f3dcd9dbb233372b61888b245c3c19fe3794bc299d7f2be55f294bb7046"
    value={24834800000000000}
  />
  <TRow
    address="1BFYggPKpVNtZmK8V7hHQvQ9FC9LAt61gH"
    currency="BTC"
    key="tableRow-23"
    txid="ead54c476870f20a3aca491fcf16b13c203f4c95008593865e210da04bdebf9d"
    value={70000}
  />
  <TRow
    address="18hh8V5fK62w13UcZEgabyHfLPc3HLBKCj"
    currency="BTC"
    key="tableRow-24"
    txid="62815ce88ee2660c1e3ae65683254fa61bcb00983e95b42097226a30ff07ed9b"
    value={556105}
  />
  <TRow
    address="1JA9dei6yMxpMWb2RWExu2pB4E3JwpMs5P"
    currency="BTC"
    key="tableRow-25"
    txid="39b1ffaac9468f737f4703cfe71bece39bae934792dd9dc59e36feb30c424d12"
    value={738510}
  />
  <TRow
    address="3757WBguTEpkDT1FMwmFXQMRNmAgWTCks8"
    currency="BTC"
    key="tableRow-26"
    txid="08f03430265a094ccca78db15eacb7e5b270f641d2a86c3991ba8c38655a054b"
    value={1500000}
  />
  <TRow
    address="1CrE4Jn5TCNw22RZq6phM9dF1Ma6Km89Ew"
    currency="BTC"
    key="tableRow-27"
    txid="08f03430265a094ccca78db15eacb7e5b270f641d2a86c3991ba8c38655a054b"
    value={1500000}
  />
  <TRow
    address="1CiFkohNFrSSwJLCrKbYzuh8T5hHi5Q1rQ"
    currency="BTC"
    key="tableRow-28"
    txid="08f03430265a094ccca78db15eacb7e5b270f641d2a86c3991ba8c38655a054b"
    value={1500000}
  />
  <TRow
    address="3422VtS7UtCvXYxoXMVp6eZupR252z85oC"
    currency="BTC"
    key="tableRow-29"
    txid="114e6bfc5ba97943e02bdd2d0ec26856048699dd53356ff73672737fee8db409"
    value={1349455}
  />
  <TRow
    address="LbTXvopZ7AE67o6bPVpQVfYLXhv52rtME2"
    currency="LTC"
    key="tableRow-30"
    txid="7b81396570855bc3ce2dee3b5551f02c9d1f8f1a3d8bf8720e7e227d8b887988"
    value={190000000}
  />
  <TRow
    address="985077b8a99b2a87a33b8b5e4da8e63ffaa0f8e6"
    currency="ETH"
    key="tableRow-31"
    txid="c905b1b9a6cc406711e55f8f4c7a4dc54bbbc0dd4e475c350fa77b937528fcb1"
    value={140000000000000000}
  />
  <TRow
    address="18PFNuXwj25Bm2obgL4YM4ACtfuEzmDFVx"
    currency="BTC"
    key="tableRow-32"
    txid="1104bb9dd49424a451f9964222ca1a2ef017cd863acfdc6ab755800e307fa16c"
    value={55635}
  />
  <TRow
    address="171YbBoD8HqyJpWUhUf4AwoWtrYHYqsiRo"
    currency="BTC"
    key="tableRow-33"
    txid="1104bb9dd49424a451f9964222ca1a2ef017cd863acfdc6ab755800e307fa16c"
    value={55635}
  />
  <TRow
    address="f42686bbc06b8309608ec4d1e46cb4329748634e"
    currency="ETH"
    key="tableRow-34"
    txid="8109e8a871ccd7faed096ce3638677dd0cdce6b3184e03b25bbb21fe58228881"
    value={1350000000000000000}
  />
  <TRow
    address="1D4eY2xAmu8LWmayWahXsn73gscEK18hDf"
    currency="BTC"
    key="tableRow-35"
    txid="b2c34a715a4d6ba0b27137a9fbc92cd54603a93929220301cf9e3e3e8fe3784b"
    value={1000000}
  />
  <TRow
    address="1Lo459z1FXLXsTkaCDPuTno2TX9EMNJS83"
    currency="BTC"
    key="tableRow-36"
    txid="b2c34a715a4d6ba0b27137a9fbc92cd54603a93929220301cf9e3e3e8fe3784b"
    value={1000000}
  />
  <TRow
    address="18Xuuvnm6JEjmGUyRFeBVAQSKabZRA3fmG"
    currency="BTC"
    key="tableRow-37"
    txid="b2c34a715a4d6ba0b27137a9fbc92cd54603a93929220301cf9e3e3e8fe3784b"
    value={1000000}
  />
  <TRow
    address="19v73cyNt8Mi1uXb8CTEX2fhELz3h4Eb8z"
    currency="BTC"
    key="tableRow-38"
    txid="b2c34a715a4d6ba0b27137a9fbc92cd54603a93929220301cf9e3e3e8fe3784b"
    value={1000000}
  />
  <TRow
    address="1BUu9qgeU2GvofVB4vJcx3oeePkiZTKBpt"
    currency="BTC"
    key="tableRow-39"
    txid="b2c34a715a4d6ba0b27137a9fbc92cd54603a93929220301cf9e3e3e8fe3784b"
    value={1000000}
  />
  <TRow
    address="175T5CLcUswxupqyFvSC6dEe6PX2bTRLWL"
    currency="BTC"
    key="tableRow-40"
    txid="b2c34a715a4d6ba0b27137a9fbc92cd54603a93929220301cf9e3e3e8fe3784b"
    value={1000000}
  />
  <TRow
    address="5f029216cb1f5f0622f1029bb7f42a1b061b5bc3"
    currency="ETH"
    key="tableRow-41"
    txid="0d4860e8e658d9734f9f21720639d50cec0d5c93a921b5b61ec0dbd0a3e8757c"
    value={1000000000000000000}
  />
</tbody>
`;
